ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -0,0 +1,218 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DomainRandomization : MonoBehaviour
+{
+    public GameObject cube;
+    public GameObject robot;
+    public GameObject table;
+
+
+    public float scaleObject = 0.1f;
+
+    public float robotMinReach = 0.2f;
+    public float robotMaxReach = 0.5f;
+
+    public int nbMaxCylinders = 5;
+    public int nbMaxSpheres = 5;
```

Comment:
We could get more variation if we defined a `maxDistactorObjects` number, and then each of those distractor objects would be randomly chosen to be either a cylinder or a sphere. Ideally, we'd have a list of "supported distractor objects types" which would contain cylinder and sphere at the moment, but that could be easily extended to add new distractor objects.